.github/ISSUE_TEMPLATE/release.md

```diff
@@ -30,9 +30,7 @@ assignees: ""
 
 ### Infrastructure provider
 
-- [ ] Cleura
 - [ ] Elastx
-- [ ] Exoscale
 - [ ] Safespring
 - [ ] UpCloud
```

Comment:
Add Azure, also do the same change for the upgrade steps below line 550.

Reply:
Done
Nice catch :) 
Thx